ecchronos-binary/src/test/behave/features/steps/ecctool_get_repairs_steps.py

```diff
@@ -75,6 +75,12 @@ def step_show_repair(context, keyspace, table):
     run_ecc_repair_status(context, ['--keyspace', keyspace, '--table', table])
     handle_repair_output(context)
 
+@when(u'we list repairs for hostid and table {keyspace}.{table}')
+def step_show_repair_with_nodeid(context, keyspace, table):
+    run_ecc_repair_status(context, ['--keyspace', keyspace, '--table', table,
+                                    '--hostid', '{0}'.format(context.host_id)])
```

Comment:
It might be good to give this value (and all other values set in environment.py) another name. Something that will enforce that this is set in the environment file. Like context.environment.host_id or similar.